### DISCUSSION TITLE
Mapping SDF-file molecules with a corresponding SMILES

### DISCUSSION THREAD

**Discussion by YoavShamir5** [rdkit/rdkit#7819] [Q&A]
Hi, does Rdkit have a built-in method to read a molecules with 3D coordinates in SDF format, and then get mapping between atoms in the conformer, and atoms in a corresponding SMILES representation? Thanks!

**Comment by ptosco** [comment]
@YoavShamir5 After calling `Chem.MolToSmiles` the `_smilesAtomOutputOrder` is set on the respective molecule, and it bears the mapping of each atom in the SMILES string to the atom index in the molecule.

**Reply by YoavShamir5** [reply]
Did not know this, great, thanks Paolo.
